src/language-js/printer-estree.js

```diff
@@ -52,7 +52,6 @@ const {
     ifBreak,
     breakParent,
     lineSuffixBoundary,
-    addAlignmentToDoc,
```

Comment:
if you want to leave the other part commented out instead of deleted, then i'd comment this out as well